app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationHeader/integrationHeader.jsx

```diff
@@ -42,6 +43,26 @@ export const IntegrationHeader = (props) => {
     breadcrumbs,
   } = props;
 
+  const { linkToDocumentation = '' } = details;
+
+  const integrationDescription = PLUGIN_DESCRIPTIONS_MAP[name] ? (
+    <>
+      {PLUGIN_DESCRIPTIONS_MAP[name]}{' '}
+      {Parser(
+        formatMessage(messages.linkToDocumentation, {
+          a: (link) => createExternalLink(link, linkToDocumentation),
+        }),
+      )}
+    </>
+  ) : (
+    <>
+      {details.description && Parser(details.description)} Link to{' '}
+      <a target="_blank" rel="noreferrer noopener" href={linkToDocumentation}>
+        Documentation
+      </a>
```

Comment:
Why not to reuse the block with docs link (https://github.com/reportportal/service-ui/pull/3577/files#diff-658180506b1b9693b5ba948e1bcd52be9f9c0dc1abcaa0cb9540e39bf09e7d63R51-R55) here as well?

Reply:
The idea was the following. If we don't store the plugin description at service-ui, it will be displayed only in English. If I reuse {Parser( formatMessage(messages.linkToDocumentation, { a: (link) => createExternalLink(link, linkToDocumentation), }), )}

and user switch the language, he will see the integration description in Eng, and Ссылка на документацию.